### PR TITLE
docs: update expo metro.config.js

### PR DESCRIPTION
**Pull request by leonardo-fc**
Added Expo specific metro.config.js instructions to MIGRATION.md and MANUAL_SETUP.md.

### CODE REVIEW

File: MANUAL_SETUP.md [hunk 1]
```diff
@@ -92,7 +92,24 @@ export default StorybookUIRoot;
 
 Update your metro config to include sbmodern in the resolverMainFields. 
 
-On expo you might need create the metro.config.js file.
+**Expo**
+
+First create the file if you don't have.
```
Comment (dannyhw) [inline MANUAL_SETUP.md]: Just a slight adjustment:

```suggestion
First create metro config file if you don't have it yet. 
```

File: MANUAL_SETUP.md [hunk 2]
```diff
@@ -92,7 +92,24 @@ export default StorybookUIRoot;
...
+```sh
+npx expo customize metro.config.js
+```
+
```
Comment (dannyhw) [inline MANUAL_SETUP.md:101]: Another small thing:

```suggestion
Then add sbmodern to the start of the `resolver.resolverMainFields` list.

```

**Comment by dannyhw** [comment]
@leonardo-fc this is a great contribution, thanks!

I made a few small suggestions, if you could apply them that would be great 🙏

**Comment by leonardo-fc** [comment]
Great.

Btw, thank you for your enormous effort 🙇‍♂️

**Review by dannyhw — APPROVED** [review]
thanks 🙇

**Pull request merged**